test.js

```diff
@@ -1,7 +1,12 @@
 import test from 'ava';
 import execa from 'execa';
 
+const shortCircuit = () => execa('./cli.js', ['--timeout', '1']);
+
 test(async t => {
 	const {stdout} = await execa('./cli.js');
 	t.true(stdout.length > 0);
 });
+
+test('should exit with 0 when online', t => t.notThrows(execa('./cli.js')));
+test('should exit with 1 when offline', t => t.throws(shortCircuit()));
```

Comment:
I generally don't like using inline arrow functions when it doesn't return anything. Makes the code harder to read.  Wrap the bodies in braces and put them on a new line.